sdk/template/azure-template/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Release History
 
-## 1.0.0-beta.1 (Unreleased)
+## 1.0.0-beta.2 (2020-09-09)
+
+* Enable master replacement link feature for cpp.
```

Comment:
Could this line say something about "Engineering System" so external customers know its not an SDK feature